examples/ui-demo/src/components/shared/CodePreviewSwitch.tsx

```diff
@@ -12,7 +12,7 @@ const CodePreviewSwitch = React.forwardRef<
 >(({ className, ...props }, ref) => (
   <SwitchPrimitives.Root
     className={cn(
-      "peer inline-flex h-[28px] w-[50px] shrink-0 cursor-pointer items-center rounded-full border-2 border-transparent transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 focus-visible:ring-offset-background disabled:cursor-not-allowed disabled:opacity-50 data-[state=checked]:bg-brand data-[state=unchecked]:bg-[#94A3B8]",
+      "peer inline-flex h-[28px] w-[50px] shrink-0 cursor-pointer items-center rounded-full border-2 border-transparent transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 focus-visible:ring-offset-background disabled:cursor-not-allowed disabled:opacity-50 data-[state=checked]:bg-[#020617] data-[state=unchecked]:bg-[#94A3B8]",
```

Comment:
nit: can we put these bg colors into the theme for the ui-demo somewhere? I ran into some inconsistencies with background colors on the toggles (eg. Email toggle vs Passkey On Signup toggle when passkeys are off)

There's also a thread about revisiting some of the bg colors of the toggles and that will make it easier to address them all

Reply:
These are hardcoded because we don't want them to change when we alter the theme from light to dark

Reply:
If we end up theming the sidebar we will refactor all the hardcoded values.